bin/cylc

```diff
@@ -524,6 +526,8 @@ if len(args) == 1:
         sys.exit(0)
     if (args[0] == '-v' or args[0] == '--version'):
         # cylc -v,--version
+        # is now an alias for "cylc version"
```

Comment:
What do you mean?   If ```cylc -v/--version``` now invokes ```cylc-version``` which generates the same output, how can there be any changed behaviour?

I suppose for continuity it might be best to leave ```cylc -v/--version``` as they were (printing version string from the main command) and just add ```cylc version``` as a separate new sub-command?

(note the new sub-command is better because you can use it do check version compatibility on a remote host very easily).